src/DotLiquid.Tests/ContextTests.cs

```diff
@@ -233,6 +233,19 @@ public void TestVariables()
             Assert.That(_context["nil"], Is.EqualTo(null));
         }
 
+#if NET6_0_OR_GREATER
+        [Test]
+        public void TestVariables_NET60()
+        {
+            var dateOnly = new DateOnly(2013, 9, 10);

```

Comment:
nit: could use named parameters to disambiguate what integer is for what. It is easy to mix up the order as all are the same type. Although the year is blatant, the month and date could be either.